sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -800,6 +1107,36 @@ export class ServiceBusManagementClient extends ServiceClient {
     return this.buildListSubscriptionsResponse(response);
   }
 
+  /**
+   * Lists existing subscriptions.
```

Comment:
The method description should clarify the difference between this method and the other  "get" operation. Same for all the "runtimeinfo" methods introduced in this PR

Reply:
Yes, I noticed.

Reply:
Added some, let me know if that's enough or if you'd like tweak.